provisions/roles/openshift/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 origin_image_registry: docker.io
 origin_image_name: openshift/origin
-origin_image_tag: v3.6.0
+origin_image_tag: v3.7.1
```

Comment:
this is new openshift origin version, can we rename the PR as "Upgrade using OpenShift version 3.7.1" ?

Reply:
that solved our current problem of not showing build logs, lets not rename it openshift upgrade, as it does not impact us much